go/go.mod

```diff
@@ -74,6 +74,7 @@ require (
 	golang.org/x/net v0.0.0-20190628185345-da137c7871d7
 	golang.org/x/sys v0.0.0-20190912141932-bc967efca4b8 // indirect
 	golang.org/x/text v0.3.2 // indirect
+	google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
```

Comment:
Oh the irony, in order to support CBOR we need this import as the gRPC error type is defined in that module as a protobuf message which happens to serialize correctly in CBOR as it has `json` annotations.